### PR TITLE
Development 3.0: selinux

### PR DESCRIPTION
**Pull request by cclerget**
**Description of the Pull Request (PR):**

This PR adds SELinux support and wrap all modules in package security with a `Configure` function.

### CODE REVIEW

**Comment by jmstover** [comment]
Re-open from `development-3.0` removal

File: src/pkg/security/apparmor/apparmor_linux.go [hunk 1]
```diff
@@ -13,19 +13,20 @@ import (
 	"os"
 )
 
-// LoadProfile write apparmor profile in /proc/self/attr/exec
-func LoadProfile(profile string) error {
+// Enabled returns if apparmor is whether enabled/supported or not
```
Comment (yhcote) [inline src/pkg/security/apparmor/apparmor_linux.go]: // Enabled() returns whether apparmor is enabled/supported or not.

File: src/pkg/security/apparmor/apparmor_linux.go [hunk 2]
```diff
@@ -13,19 +13,20 @@ import (
...
+func Enabled() bool {
 	data, err := ioutil.ReadFile("/sys/module/apparmor/parameters/enabled")
 	if err == nil {
```
Comment (yhcote) [inline src/pkg/security/apparmor/apparmor_linux.go]: ...
```
        if err == nil && len(data) > 0 && data[0] == 'Y' {
                return true
        }
        return false
}
```

**Review by yhcote — APPROVED** [review]
looks good to me

**Pull request merged**